components/Processes.vue

```diff
@@ -18,7 +18,7 @@
 			@detailsToggled="detailsToggled"
 			allowCopy>
 			<template #heading="scope"><slot name="heading" v-bind="scope" /></template>
-			<template #content-start v-if="missing"><FederationMissingNotice :missing="missing" :federation="federation" /></template>
+			<template #content-start v-if="missing && Array.isArray(missing) && missing.length>0"><FederationMissingNotice :missing="missing" :federation="federation" /></template>
```

Comment:
```suggestion
			<template #content-start v-if="Array.isArray(missing) && missing.length > 0"><FederationMissingNotice :missing="missing" :federation="federation" /></template>
```

see https://github.com/Open-EO/openeo-vue-components/pull/102/files#r2036266968